### PR TITLE
feat: add record, autoStartMeeting, muteOnStart parameters to bbb

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
This PR add record, autoStartRecording, muteOnStart Parameters to BBB video-stream

required for https://github.com/fossasia/open-event-frontend/pull/7017

### CODE REVIEW

File: app/api/schema/video_stream.py [hunk 1]
```diff
@@ -5,9 +5,16 @@
 from app.api.helpers.utilities import dasherize
 
 
+class VideoStreamExtraOptionsSchema(JsonSchema):
+    record = fields.Boolean(default=True)
+    autoStartRecording = fields.Boolean(default=True)
```
Comment (iamareebjamal) [inline app/api/schema/video_stream.py:10]: This should be false by default

File: app/api/video_stream.py [hunk 2]
```diff
@@ -72,19 +72,21 @@ def join_stream(stream_id: int):
...
 
     params = dict(
-        record=True,
         name=stream.name,
         meetingID=stream.extra['response']['meetingID'],
         moderatorPW=stream.extra['response']['moderatorPW'],
         attendeePW=stream.extra['response']['attendeePW'],
+        record=stream.extra['options']['record'],
+        autoStartRecording=stream.extra['options']['autoStartRecording'],
+        muteOnStart=stream.extra['options']['muteOnStart'],
```
Comment (iamareebjamal) [inline app/api/video_stream.py:81]: Will crash the server if options is null

File: app/api/video_stream.py [hunk 3]
```diff
@@ -116,13 +118,22 @@ def create_bbb_meeting(channel, data):
...
+            muteOnStart=data['extra']['options']['muteOnStart'],
+        ),
+    )
 
     if not (res.success and res.data):
         logger.error('Error creating BBB Meeting: %s', res)
         raise UnprocessableEntityError('', 'Cannot create Meeting on BigBlueButton')
 
-    data['extra'] = res.data
+    data['extra']['response'] = res.data['response']
```
Comment (iamareebjamal) [inline app/api/video_stream.py:136]: Will crash if extra is null

File: app/api/video_stream.py [hunk 4]
```diff
@@ -72,19 +72,21 @@ def join_stream(stream_id: int):
...
+        muteOnStart=stream.extra['options']['muteOnStart'],
     )
 
     channel = stream.channel
     bbb = BigBlueButton(channel.api_url, channel.api_key)
     result = bbb.request('create', params)
 
     if result.success and result.data:
-        stream.extra = result.data
+        stream.extra['response'] = result.data['response']
```
Comment (iamareebjamal) [inline app/api/video_stream.py:89]: Will crash if extra is null

File: app/models/video_stream.py [hunk 5]
```diff
@@ -1,5 +1,5 @@
 from flask_jwt_extended import current_user
-from sqlalchemy import or_
+from sqlalchemy import event, or_
```
Comment (iamareebjamal) [inline app/models/video_stream.py:2]: revert

File: app/models/video_stream.py [hunk 6]
```diff
@@ -82,9 +82,7 @@ def user_is_moderator(self):
         user = current_user
         if user.is_staff or has_access('is_coorganizer', event_id=self._event_id):
             return True
-        return user.email in list(
```
Comment (sachinchauhan2889) [inline app/models/video_stream.py:85]: done by precommit.

**Pull request merged**